src/dbos-executor.ts

```diff
@@ -339,22 +366,47 @@ export class DBOSExecutor {
     const wCtxt: WorkflowContextImpl = new WorkflowContextImpl(this, params.parentCtx, workflowUUID, wConfig, wf.name, presetUUID);
     wCtxt.span.setAttributes({ args: JSON.stringify(args) }); // TODO enforce skipLogging & request for hashing
 
+    let executorID: string = "local";
+    if (wCtxt.request.headers && wCtxt.request.headers[DBOSExecutorIDHeader]) {
+      executorID = wCtxt.request.headers[DBOSExecutorIDHeader] as string;
+    }
+    const bufStatus: BufferedStatus = {
+      workflowUUID: workflowUUID,
+      status: StatusString.PENDING,
+      name: wf.name,
+      authenticatedUser: wCtxt.authenticatedUser,
+      output: undefined,
```

Comment:
We have output and error fields in bufstatus, but always pass output and error separately

Reply:
Changed to `WorkflowStatusInternal`